node-fs-writefile/cp.js

```diff
@@ -0,0 +1,14 @@
+import fs from 'node:fs/promises';
+
+const [, , f1, f2] = process.argv;
+
+async function copyToFile() {
+  const f1Text = await fs.readFile(f1, 'utf8');
```

Comment:
By passing an encoding, you make this function so that it can only copy text files. If you leave off the encoding, it will copy any file.